src/LRS.js

```diff
@@ -456,12 +449,34 @@ TinCan client library
                     statementId: stmtId
                 }
             };
+            if (cfg.params.attachments) {
+                requestCfg.params.attachments = true;
+            }
             if (typeof cfg.callback !== "undefined") {
                 callbackWrapper = function (err, xhr) {
-                    var result = xhr;
+                    var result = xhr.responseText,
+                        boundary,
+                        section,
+                        statement,
+                        i;
 
                     if (err === null) {
-                        result = TinCan.Statement.fromJSON(xhr.responseText);
+                        if (! cfg.params.attachments) {
+                            result = TinCan.Statement.fromJSON(result);
+                        }
+                        else {
+                            boundary = xhr.getResponseHeader("Content-Type").split("boundary=")[1];
+                            result = result.split("--" + boundary + "--")[0].split("--" + boundary);
+
+                            statement = JSON.parse(result[1].split("\r\n\r\n")[1]);
+
+                            for (i = 2; i < result.length; i += 1) {
+                                section = result[i].split("\r\n\r\n");
+                                statement = TinCan.LRS.prototype._assignAttachmentContent({statements: [statement]}, section[0].split("\r\n"), section[1].trim()).statements[0];
```

Comment:
Shouldn't call this method on the prototype without doing so through the instance itself. It should be made either local and not available externally, "static" on the "class", or make the instance available likely via a closure and call it as an instance method.